src/duct/__main__.py

```diff
@@ -17,6 +16,17 @@
 from . import __version__
 
 ENV_PREFIXES = ("PBS_", "SLURM_", "OSG")
+STDOUT_SUFFIX = "stdout"
+STDERR_SUFFIX = "stderr"
+USAGE_SUFFIX = "usage.json"
+INFO_SUFFIX = "info.json"
+
+SUFFIXES = {
```

Comment:
An alternative I just thought of: Add a `LogFile` (or some other name) enum with `STDOUT`, `STDERR`, `USAGE`, and `INFO` variants plus a method for getting an instance's suffix.

Reply:
How is that different than a dataclass with __iter__?

Reply:
Your `Suffix` dataclass only had once meaningful instance, which contained one field per logfile.  My suggestion is to create an enum with four instances, one instance per logfile.  This enum could then be used to replace the `stdout` etc. fields of `LogPaths` with a `paths: dict[LogFile, str]`, among other changes.

Reply:
I decided to leave with the SUFFIX dict.

2 alt implementations from side chats (just for note taking)

```python
class LogSuffix(Enum):
    STDOUT = 1
    STDERR = 2
    USAGE = 3
    INFO = 4
 
    def suffix(self) -> str:
        if self is LogSuffix.STDOUT:
            return "stdout"
        elif # etc.
```

```python
class LogSuffix(Enum):
    STDOUT = "stdout"
    STDERR = "stderr"
    USAGE = "usage.json"
    INFO = "info.json"


@dataclass
class LogPaths:
    paths: dict[LogSuffix, str]
    prefix: str

    @classmethod
    def create(cls, output_prefix: str, pid: None | int = None) -> LogPaths:
        datetime_filesafe = datetime.now().strftime("%Y.%m.%dT%H.%M.%S")
        formatted_prefix = output_prefix.format(
            pid=pid, datetime_filesafe=datetime_filesafe
        )
        paths = {suffix: f"{formatted_prefix}{suffix.value}" for suffix in LogSuffix}
        return cls(paths, formatted_prefix)
```